pkg/services/ngalert/eval/eval.go

```diff
@@ -7,14 +7,21 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/grafana/grafana/pkg/setting"
+
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
 	"github.com/grafana/grafana/pkg/expr"
 )
 
 const alertingEvaluationTimeout = 30 * time.Second
 
+type Evaluator struct {
+	Cfg *setting.Cfg
+}
+
 // invalidEvalResultFormatError is an error for invalid format of the alert definition evaluation results.
+
```

Comment:
That newline looks out of place.
```suggestion
// invalidEvalResultFormatError is an error for invalid format of the alert definition evaluation results.
```